techlibs/xilinx/xcu_brams.txt

```diff
@@ -0,0 +1,105 @@
+
```

Comment:
This file is identical to `xc7_brams.txt`, can't we just reuse it?

Reply:
Done

Reply:
Done